### PR TITLE
setting-up.md: Fix tracking of upstream repository instructions #8241

### LINKED ISSUE #8241: setting-up.md: Fix tracking of upstream repository instructions
For new contributors while setting up they might face difficulty before executing this command below.

(setting-up.md)

img_20171223_154506

The person needs to do `git fetch upstream` before setting your master branch to track the upstream's master branch as the branches of upstream haven't been fetched before setting it to be tracked

### PR DESCRIPTION
**Pull request by kevinLamKB**
Fixes #8241 

**Outline of Solution**
Added the missing line in setting-up.md doc

### CODE REVIEW

File: docs/setting-up.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 1. Set your `master` branch to track the main repo's `master` branch.
    ```sh
    git checkout master
+   git fetch upstream
```
Comment (wkurniawan07) [inline docs/setting-up.md]: This should be on the previous step as the previous step is essentially setting up the remote itself. This step can then be verified with `git branch -r`.

**Comment by kevinLamKB** [comment]
@wkurniawan07 thank you for the insights! i've made the necessary adjustments as requested, please check.

File: docs/setting-up.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
     upstream        https://github.com/TEAMMATES/teammates.git (fetch)
     upstream        https://github.com/TEAMMATES/teammates.git (push)
     ```
+1. Retrieve remote-tracking branches from the upstream repo.
```
Comment (wkurniawan07) [inline docs/setting-up.md]: This works, but you can combine it with the previous step instead. That way there's no need for `git remote -v` anymore.

File: docs/setting-up.md [hunk 3]
```diff
@@ -30,6 +30,14 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
...
+   ```sh
+   git fetch upstream
+   ```
+   **Verification:** Use the command `git branch -r` and the following lines should be part of the output:
+   ```
+   upstream/master
```
Comment (wkurniawan07) [inline docs/setting-up.md]: Can add `upstream/release` also.

**Comment by kevinLamKB** [comment]
Alright, I've updated the docs as recommended, please check once again, thank you!

**Pull request merged**